Controller/GalleryAdminController.php

```diff
@@ -66,4 +62,27 @@ public function listAction(Request $request = null)
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ));
     }
+
+    /**
+     * Sets the admin form theme to form view. Used for compatibility between Symfony versions.
+     *
+     * @param FormView $formView
+     * @param string   $theme
+     */
+    private function setFormTheme(FormView $formView, $theme)
+    {
+        $twig = $this->get('twig');
+
+        try {
+            $twig
+                ->getRuntime('Symfony\Bridge\Twig\Form\TwigRenderer')
+                ->setTheme($formView, $theme);
+        } catch (\Twig_Error_Runtime $e) {
```

Comment:
I know we use that somewhere else, but IMO [there are less violent methods for detected twig bridge 3.2](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Tests/Controller/CRUDControllerTest.php#L194), and they should be preferred, because it might also be more accurate: what if a `\Twig_Error_Runtime` is thrown in `^3.2` for a totally unrelated reason?